### PR TITLE
Redesign organization card in settings

### PR DESCRIPTION
**Pull request by manuthecoder**
See #6607

### CODE REVIEW

**Comment by manu-gurudath** [comment]
Here's what it looks like so far: 

image
image

File: app/views/events/settings/_details.html.erb
Comment (sampoder) [inline app/views/events/settings/_details.html.erb]: Let's use the `photo` or `photo-fill` icons here. The edit one isn't super clear for non-text inputs imo.

File: app/views/events/settings/_details.html.erb
Comment (sampoder) [inline app/views/events/settings/_details.html.erb]: Would it be possible to make these two inputs the same height?

**Review by sampoder — COMMENTED** [review]
This looks incredible; thank you so much for putting this together! Two quick comments.

File: app/assets/stylesheets/application.scss
Comment (malted) [inline app/assets/stylesheets/application.scss]: For consistency, what if this was named `event_settings`?

**Comment by malted** [comment]
There seems to be a layout issue on mobile - can we make the inputs wrap & stick a `ml-3` on the button?
image

**Comment by malted** [comment]
This new card looks amazing!

**Review by sampoder — APPROVED** [review]
LGTM, thank you so much for this! Once this is merged, it'd be sweet to post about this publicly

**Comment by manu-gurudath** [comment]
Cool! How would I do that?

**Comment by sampoder** [comment]
Maybe just a post in #hcb with a screenshot or a #ship post?

**Pull request merged**
